aya-build/src/lib.rs

```diff
@@ -147,3 +147,21 @@ pub fn build_ebpf(packages: impl IntoIterator<Item = Package>) -> Result<()> {
     }
     Ok(())
 }
+
+#[derive(Default)]
+pub enum Toolchain<'a> {
+    /// Use the latest nightly toolchain to compile to eBPF kernel, e.g. `+nightly`.
```

Comment:
s/e.g./i.e./

e.g. = exempli gratia = for example
i.e. = id est = that is

this should be i.e., for custom e.g. is fine